src/web/components/Img.tsx

```diff
@@ -57,8 +68,25 @@ const buildSizesString = (role: RoleType, isMainMedia: boolean): string => {
         case 'thumbnail':
             return '140px';
         case 'immersive':
+            // Immersive MainMedia elements fill the height of the viewport, meaning
+            // on mobile devices even though the viewport width is small, we'll need
+            // a larger image to maintain quality. Deciding which width image to ask
+            // for is difficult because aspect ratios and orientations between
+            // different devices varies so the values given below are best estimates
+            // based on the most popular configurations
+
+            // Immersive body images stretch the full viewport width below wide,
+            // but do not stretch beyond 1300px after that.
             return isMainMedia
-                ? '100vw'
+                ? `
+                    (min-width: ${breakpoints.leftCol}px) 1600px,
+                    (min-width: ${breakpoints.desktop}px) 1900px,
+                    (min-width: ${breakpoints.tablet}px) 1900px,
+                    (min-width: ${breakpoints.phablet}px) 1600px,
+                    (min-width: ${breakpoints.mobileLandscape}px) 1300px,
+                    (min-width: ${breakpoints.mobileMedium}px) 900px,
+                    1300px
+                `
```

Comment:
```suggestion
                ? `
                    (orientation: portrait) 167vw,
                    100vw
                `
```

Reply:
Just to be clear, I think `(orientation: portrait) 167vw` above should read as `(orientation: portrait) 167vh` (`vh`) so I'm leaving this commit suggestion alone and I'll push this change up manually

Reply:
Just to note that while it’s definitely true that most main medias are 5:3, there is **nothing** stopping them being any ratio. As I understand, this (`167vh`) will control the number of steps, so if it’s gonna be off, nothing terrible will happen. Still, curious what [Kylie’s](https://www.theguardian.com/music/2020/nov/07/kylie-minogue-its-time-to-dress-in-sequins-and-glitter-through-the-darkness) gonna look like.